angr/procedures/win32/heap.py

```diff
@@ -1,17 +1,26 @@
 import angr
 
+
 class GetProcessHeap(angr.SimProcedure):
     def run(self):
-        return 1 # fake heap handle
+        return 1  # fake heap handle
+
 
 class HeapCreate(angr.SimProcedure):
     def run(self, flOptions, dwInitialSize, dwMaximumSize):
-        return 1 # still a fake heap handle
+        return 1  # still a fake heap handle
+
 
 class HeapAlloc(angr.SimProcedure):
     def run(self, HeapHandle, Flags, Size):
         return self.state.heap._malloc(Size)
 
+
 class GlobalAlloc(HeapAlloc):
     def run(self, Flags, Size):
         return super(GlobalAlloc, self).run(1, Flags, Size)
+
+
+class HeapFree(angr.SimProcedure):
+    def run(self, HeapHandle, Flags, lpMem): #pylint:disable=arguments-differ
+        return 1
```

Comment:
need a newline at end of file

Reply:
Also I think there's an actual function you can call to do frees in the heap plugin